deps/kspec.jl

```diff
@@ -83,6 +83,7 @@ function installkernel(name::AbstractString, julia_options::AbstractString...;
             "display_name" => name * " " * Base.VERSION_STRING * debugdesc,
             "language" => "julia",
             "env" => env,
+            "interrupt_mode" => "message",
```

Comment:
Seems like this should be `Sys.iswindows() ? "message" : "signal"`, since SIGINT is probably still more reliable on non-Windows systems.

Reply:
I don't really care much, though I don't see how there would be a difference in reliability. In the end, the same mechanism is used to interrupt the running `execute_request`.

Reply:
See below.  In the current Julia, the tasks are "green threads" that run cooperatively, so the task listening to the control socket may be starved of cycles.   sigint circumvents this.